### PR TITLE
Ephedrine small nerf

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request

Nerfs Ephedrine so you can get hand spasms when using 100% purity Ephedrine

## Why It's Good For The Game
Pure Ephedrine in its current state is really strong, easy to make and has negligible downsides, This aims to tackle that
## Changelog
:cl:
balance: Makes it so Ephedrine spasms have a 10 * (1.5 - purity)% chance per second to happen, Adding a downside to pure Ephedrine 
/:cl:

### CODE REVIEW

**Comment by Pickle-Coding** [comment]
I suggest including "per second" in the changelog to make it less confusing.

**Comment by starrm4nn** [comment]
> I suggest including "per second" in the changelog to make it less confusing.
alright thanks for the suggestion

**Comment by Helg2** [comment]
So it will be 50% of dropping something in a tick even with pure ephedrine? That's pretty dumb, instead of making it lets say harder to obtain you just make it useless for most cases

**Comment by blackdav123** [comment]
> So it will be 50% of dropping something in a second even with pure ephedrine? That's pretty dumb, instead of making it lets say harder to obtain you just make it useless for most cases Or, wait. 1.5 - 100% = 0, so i dont quite get it.

wouldnt this make it a 5-15% chance of dropping items rather than a 0-10% chance based on purity?

**Comment by starrm4nn** [comment]
> So it will be 50% of dropping something in a second even with pure ephedrine? That's pretty dumb, instead of making it lets say harder to obtain you just make it useless for most cases Or, wait. 1.5 - 100% = 0, so i dont quite get it.

100% purity in the calculation is 1
so its 1.5 - 1 at 100% purity
so it becomes a 5% drop chance instead of 0

**Comment by starrm4nn** [comment]
> So it will be 50% of dropping something in a second even with pure ephedrine? That's pretty dumb, instead of making it lets say harder to obtain you just make it useless for most cases Or, wait. 1.5 - 100% = 0, so i dont quite get it.

Meth doesn't have the item drop downside

**Comment by moocowswag** [comment]
Indirect hulk buff since hulks dont need to hold items and can continue to enjoy the speed increase while non hulks cant necessarily use this chem effectively anymore

**Comment by Jackal-boop** [comment]
epedrine in its current state is way too powerful and needs a pegging down. every shift i see chemists with it in their blood dabbing around everyone and starting fights

**Comment by carlarctg** [comment]
The 13 downreddits are so fucking funny. People going NOOOO MY FRAG CHEM!! NOO!! Without any real argument

The hulk thing is irrelevant. Hulks are well known to be stupidly busted and shouldn't really be relevant in this way to chem balance, and while Hulk punches are tremendous, they aren't even as strong as a spear or fire axe. Using Ephedrine is still a problem as well since you may need utility items during combat, such as medipens, tools, teleporters, etc.

**Comment by moocowswag** [comment]
>The hulk thing is irrelevant.

It will always be relevant because balance is relative, if all ways to counter knockdowns are nerfed (most likely a cumulative change over a long period of time) then that is small incremental buffs to hulk.

> they aren`t even as strong as a spear or fire axe.

would you rather fight a hulk who has ephedrine or a moth who has ephedrine and a fireaxe



and to say hulk punches are not as strong as a fire axe is false, while they do less upfront damage and do not come with bloodloss they make up for it by ignoring armor and having a massive wound bonus allowing easy access to limb breaking which can disable you and stop you from effectively fighting back or running away.





Again, I think a higher metabolization rate is the most balance-able approach since it forces more maintenance on your supplies and therefore downtime. Not to mention this sort of nerf isnt bypassed by no-drop implants or just not being reliant on held items or items that can be dropped 

This nerf doesnt effect anyone who uses
krav maga, sleeping carp, no drop implant, me dropping circuit bombs, hulk, ability casters (wiz/heretic/ling sting/gene) stuck items (armblade/some chap null rods) 

Notice the most accessible thing here is hulk which also happens to be even better stun combat resistance than ephedrine but can also make use of the movement speed.


TLDR this nerf can be gamed, I already dont care about full purity ephedrine because I only use it for the stupid mobility it gives and I dont even use my hands I just hit and run with bombs and its absurd that the amount of downtime when I use this is basically zero because a single beaker will last me an entire round even when im using it semi-often.

**Comment by carlarctg** [comment]
The whole point of Ephedrine is that it increases your speed more than Meth but causes hand cramps. Using it with Hulk punches, Krav Maga, Carp, nodrop items, is not an 'exploit', it's just what one prepares for ephedrine?? It'd be like saying that putting meth and mannitol in the same pill is an exploit that bypasses the downside. It's not, you're just adapting your strategy.

Your circuit bombs are something else entirely, and I'm not sure why youre bringing them up on a chem nerf PR. Bombs are just busted in general, we know that.

Again if you're worried about hulk, make them unable to get movement speed buffs. It's not that I disagree, ephedrine hulks are cancer, I just fail to see the relevancy of hulk walance on an unrelated PR.

**Comment by carlarctg** [comment]
It doesn't make it useless, it makes it have downsides. If you want a general stimulant make meth. If you want to go fas, make ephedrine.

**Comment by moocowswag** [comment]
> is not an 'exploit'

Nobody said it was , my point clearly flew over your head if you arnt even responding to my suggestion for metabolization rate being increased.

**Comment by Yttriums** [comment]
Worth noting that this is also nerfing the most easily distributable stimulant chem from botany. I enjoyed distributing a lot of coffee robusta/ephedrine as botanist because it puts people on a level playing field with chem powergamers and also nerfs stuns, two things which (in my opinion) make the round more fun. Drop chance is a bad enough nerf to make people not use coffee robusta. Meth + mannitol plants are possible, but it's a shift's worth of effort. 

I think this will create an environment where combat stims are for chemists and depraved powergamers only, as they will be the ones using meta recipes to quickly produce meth + mannitol + etc. Those players will then dab even harder on everyone else.

**Comment by zoomachina** [comment]
> The whole point of Ephedrine is that it increases your speed more than Meth but causes hand cramps. Using it with Hulk punches, Krav Maga, Carp, nodrop items, is not an 'exploit', it's just what one prepares for ephedrine?? It'd be like saying that putting meth and mannitol in the same pill is an exploit that bypasses the downside. It's not, you're just adapting your strategy.

as a matter of fact, meth is much faster than ephedrine, and ephedrine's speedup is the same as nuka cola; also, munching timegate-locked, high-risk, traitor items or items from other departments really isn't a good "solution" to make ephedrine useful (after this nerf); the actual useful effect of ephedrine is immunity to baton 2-click game over, which this pr just kills

**Comment by moocowswag** [comment]
> Chem gamers are really crawling out of thr wood work for this huh

I dont think ive seen a single chemist gamer respond to the pr yet

just me saying it needs a metabolization hit and botany player saying its a nerf to non chemists.

**Comment by Yttriums** [comment]
> Chem gamers are really crawling out of thr wood work for this huh

Chem gamers don't care about this, they just have to memorize a slightly longer list of steps to get their speed beakers. It barely punishes them.

It punishes botany much harder for no reason.

**Comment by wesoda25** [comment]
gd i hate ephedrine so much. Purity enabling you to bypass the intended drawbacks of the chem is SO dumb. the POINT is its a getaway chem you dont NEED hands. good pr 👏 👏

**Comment by zoomachina** [comment]
> > The whole point of Ephedrine is that it increases your speed more than Meth
> 
> HUH
(it's not true)

**Comment by Samag5** [comment]
I suggest adding a chance to drop item that you took from backpack/belt/etc. Cuz you can just counter that small nerf using hotkeys.

**Comment by improvedname** [comment]
ephedrine isnt that great tbh its just baton knockdown immunitt

**Comment by starrm4nn** [comment]
> ephedrine isnt that great tbh its just baton knockdown immunitt

and a 50% to movement speed?

**Comment by starrm4nn** [comment]
so over...

**Comment by JohnFulpWillard** [comment]
i dont want to read all this but serxule downvoted so
